tasks/started/github-mcp-integration.md

```diff
@@ -0,0 +1,123 @@
+  - Verify it updates the AI agent instructions with GitHub operations
+  - Confirm idempotent behavior (doesn't overwrite existing files without --force)
+
+- Test bootstrap script with --without-github-mcp flag:
+  - Verify it doesn't include GitHub MCP documentation and workflow sections
+
+- Test the GitHub MCP integration using this task:
+  - Create branch for this task following naming convention
+  - Make required implementation changes
+  - Create PR and update task metadata
+  - Verify task can remain in "started" folder during PR review
+  - Verify task file is updated with PR information
+
+- Test with both new and existing projects:
+  - Verify the GitHub MCP integration works when bootstrapping a new project
+  - Verify adding or removing GitHub MCP support to an existing project works correctly with the --force flag
+
+## Evidence of Completion
+- [x] Command output or logs demonstrating completion: All tests pass successfully with the updated test script
+- [x] Path to created/modified files:
+  - Added `/docs/agentic/github-mcp-guide.md` with GitHub MCP operations documentation
+  - Updated `/docs/agentic/ai-readme.md` with GitHub operations workflow
+  - Modified `agentic-bootstrap.sh` to add GitHub MCP integration by default
+  - Updated `test_bootstrap.sh` to test GitHub MCP integration features
+  - Updated `docs/components/bootstrap-script.md` to document the new feature
+  - Updated `docs/components/test-bootstrap-script.md` to reflect the enhanced tests
+- [x] Summary of changes made:
+  1. Created comprehensive GitHub MCP operations guide with command templates
+  2. Updated AI agent instructions with Git/GitHub workflow procedures
+  3. Added GitHub MCP integration to bootstrap script (enabled by default)
+  4. Implemented `--without-github-mcp` flag to disable GitHub integration
+  5. Enhanced test script with additional tests for GitHub MCP features
+  6. Updated documentation to reflect all changes
+  7. Maintained idempotent behavior with the new features
+  8. Updated bootstrap script help text with new flag details
+- [x] Branch name created for this task: `task/github-mcp-integration`
+- [x] PR number and URL for this task: [PR #1](https://github.com/chris-sanders/ai-bootstrap/pull/1)
+
+## Notes
+- This task only configures workflow instructions for GitHub MCP, it does not configure the Claude Code MCP server itself, as that's handled at the user level.
+- The flag follows the same convention as the existing `--with-adr` flag, but inverse logic as it disables a feature that's on by default.
+- For Phase 4 (Future Work), a placeholder task for GitHub issues integration should be created in `/tasks/backlog/` after this task is completed, but is not part of this immediate implementation.
+- Task status flow: Tasks stay in "started" folder until PR is merged, even after implementation is complete. This ensures AI agents can find and continue work on tasks with open PRs by looking in the "started" folder.
```

Comment:
These are good where they are but would also be good to include in the Pull Request comment so a reviewer doesn't have to come find this file to get an overvoew of this information on Github.